src/uu/hashsum/src/hashsum.rs

```diff
@@ -24,13 +24,16 @@ use std::iter;
 use std::num::ParseIntError;
 use std::path::Path;
 use uucore::error::{FromIo, UError, UResult};
+use uucore::{format_usage, help_about, help_usage};
 use uucore::sum::{
     Blake2b, Blake3, Digest, DigestWriter, Md5, Sha1, Sha224, Sha256, Sha384, Sha3_224, Sha3_256,
     Sha3_384, Sha3_512, Sha512, Shake128, Shake256,
 };
 use uucore::{crash, display::Quotable, show_warning};
 
 const NAME: &str = "hashsum";
+const ABOUT: &str = help_about!("hashsum.md");
```

Comment:
please run rustfmt

Reply:
I did run `cargo fmt`

Reply:
the CI disagrees :) 
could you please try again? (and update to the most recent version of rustfmt)

Reply:
Sorry, that's my fault. The style violation is on the commit [hashsum: merge imports](https://github.com/uutils/coreutils/pull/4563/commits/62e5c5a7554d94e03270addd7f3a4020244dbfe1).